tests/move_only_function/test_noexcept.cpp

```diff
@@ -7,6 +7,7 @@ constexpr auto nothrow_call = [](move_only_function<int() noexcept> f)
 
 template<class... T> struct overload : T...
 {
+    constexpr overload(const T &... t) : T(t)... {}
```

Comment:
How about adding a deduction guide
```cpp
template<class... T>
overload(T...) -> overload<T...>;
```
after the class body as a workaround?

Reply:
This is probably a bug in clang/clang-cl (observed in versions 14 and 15). Clang gives the following error:

~~~
error: no viable constructor or deduction guide for deduction of template arguments of 'overload'
~~~

Apparrently, it does not consider the member-wise struct initializer when deducinf the template arguments. GCC handles the original code just fine.

I wrapped the workarounf in #ifdef __clang__.

Reply:
As the error message said, missing the (implicitly generated) deduction guide was the problem.